flaskbb/cli/main.py

```diff
@@ -615,6 +616,14 @@ def generate_config(development, output, force):
         click.style("Mail Admin Email", fg="magenta"),
         default=default_conf.get("mail_admin_address"))
 
+    click.secho("Optional filepath to load a logging configuration file from. "
```

Comment:
Should I add the other new log config options here as well?

Reply:
I think that would be an overkill for our guided setup - just the file is fine imo

Reply:
Alright, I'll update the docs to call these out then.